JHUGenLexicon/interface/JHUGenLexiconCouplings.h

```diff
@@ -9,7 +9,7 @@
 #define DEFVAL_SW 0.23119 //Actually is SW^2
 #define DEFVAL_LAMBDA_VI 10000.
 #define DEFVAL_ALPHA .0072973525693 //Used to set e^2
-#define DEFVAL_VEV_LAM .0024622 //Vacuum Expectation Value GeV / Lambda Warsaw Basis
+#define DEFVAL_VEV_LAM 0.0606242884 //Vacuum Expectation Value GeV / Lambda Warsaw Basis
```

Comment:
What is the advantage of using this value? How do we make sure this change does not affect what is used in these lines? https://github.com/JHUGen/JHUGen/blob/8c285ec34b7f3f4c68a281bee0cecb3fdc442d3c/JHUGenLexicon/src/JHUGenLexiconTranslator.cc#L249-L319

Reply:
vev_lam in translation is (vev/lam)^2. The default value that we had set earlier is vev/(lam)^2 which is not the proper default. The translation needs to be proportional to vev^2 as well